server/controllers/swatchController.js

```diff
@@ -51,7 +51,48 @@ swatchController.getPalette = async (req, res, next) => {
     return next({
       log: error,
       status: 400,
-      message: { err: 'An error occurred in `swatchController.getPalette`' },
+      message: { err: "An error occurred in `swatchController.getPalette`" },
     });
   }
 };
+
+swatchController.saveImage = async (req, res, next) => {
+  try {
+    console.log("running swatchController.saveImage");
+    const url = req.body.url;
+    console.log("url:", url);
+
+    const queryForImage = `
+    SELECT id
+    FROM images
+    WHERE url = '${url}'
+    `;
+
+    const queryToInsert = `
+    INSERT INTO images (url)
+    VALUES ('${url}')
+    ON CONFLICT DO NOTHING
+    RETURNING id
+    `;
+
```

Comment:
Can we use parameterized query instead? See Eileen's PR [here](https://github.com/Swatch-My-Image/Swatch-My-Image/pull/12) for some examples. This is very vulnerable to SQL-injection attacks.